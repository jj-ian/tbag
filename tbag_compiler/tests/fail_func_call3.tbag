true {
        int x = gcd("yo", 14);
	endgame;
}

