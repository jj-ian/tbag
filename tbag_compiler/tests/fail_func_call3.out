Fatal error: exception Failure("Function gcd does
                            not exist with the given parameters.")
